simulators/ethereum/engine/helper.go

```diff
@@ -24,7 +24,7 @@ import (
 // default timeout for RPC calls
 var rpcTimeout = 10 * time.Second
 
-// TestClient is the environment of a single test.
+// TestClient is 00the environment of a single test.
```

Comment:
00? are you using i3?

Reply:
That was my bad.. And yes, I use i3. But how is that typo attributable to i3???